packages/browser/src/ClientAuthentication.ts

```diff
@@ -118,33 +120,14 @@ export default class ClientAuthentication {
       return null;
     }
     const sessionInfo = await this.sessionInfoManager.get(currentSessionId);
```

Comment:
Let's also make sure that any sessionInfo that would've contained an ID Token strips it here, such that `sessionInfo` absolutely does not have an ID Token in it.

Reply:
Good point, 73b077e08cd762496aafbb937b50572ecb76ae8d removes the `idToken` property from the session info type, and enforces that it isn't  looked for in storage.